.pre-commit-config.yaml

```diff
@@ -15,11 +15,11 @@ repos:
     rev: v2.2.2
     hooks:
     -   id: codespell
-        args: ["--count --ignore-words-list=ans,deriver,inout,packag --skip=\"*.js,*WordLists.swift,.git,Carthage,.build,build\""]
+        args: [--count --ignore-words-list="ans,deriver,inout,packag" --skip="*.js,*WordLists.swift,.git,Carthage,.build,build"]
 # SwiftLint is commented out until we fix all SwiftLint warnings and errors.
 # https://github.com/web3swift-team/web3swift/pull/756
-#-   repo: https://github.com/realm/SwiftLint
-#    rev: 0.50.3
-#    hooks:
-#    -   id: swiftlint
-#        args: [--fix, Sources, Tests]
+-   repo: https://github.com/realm/SwiftLint
```

Comment:
If I understood @yaroslavyaroslav correctly here, you might be able to use `swiftlint` directly: https://github.com/web3swift-team/web3swift/pull/690#discussion_r1036997551

Reply:
Looks like it, but your comment is more relevant to GitHub Actions CI than to `pre-commit-config`. Not all GitHub Action runners have `swiftlint` by default. Ubuntu for example doesn't have it.

`pre-commit-config` is primarily for local use, or at least it was configured that way. This configuration ensures that you'll have `swiftlint` installed.

Reply:
https://pre-commit.ci was created to show that pre-commit is not just for local use.

Reply:
Yea this is why I was concerned when first commeting. Obvisouly, I'm not against as much checking as possible for PRs, I'm just not a big fan of too much inconvenience locally.

Reply:
> Yea this is why I was concerned when first commeting. Obvisouly, I'm not against as much checking as possible for PRs, I'm just not a big fan of too much inconvenience locally.

Yea this is why I was concerned when first commeting. Obvisouly, I'm not against as much checking as possible for PRs, I'm just not a big fan of too much inconvenience locally.